src/bespokelabs/curator/request_processor/base_request_processor.py

```diff
@@ -398,7 +399,10 @@ def create_dataset_files(
                         f"Some requests do not have responses and require_all_responses is True."
                     )
 
-        return Dataset.from_file(dataset_file)
+        d = Dataset.from_file(dataset_file)
```

Comment:
Why do we need to sort?

Reply:
If I sent requests [req1, req2, req3] (say through a dataset), the ordering of the responses can be anything right now. And varies with each run.
It's a bit confusing, and most people expect the responses to be in the order they sent the queries.